src/main/java/frc/robot/Constants.java

```diff
@@ -34,12 +23,44 @@
 import frc.robot.subsystems.Elevator;
 
 public final class Constants {
+  public static class MotorIDConstants {
+    public static final int kCLimberMotorLeader = 12;
+    public static final int kCLimberMotorFollower = 13;
+    public static final int kScorerMotor = 9;
+    public static final int kElevatorMotor1 = 10;
+    public static final int kElevatorMotor2 = 11;
+    public static final int kIntakeMotor = 9;
+  }
+
+  public static class ClimberConstants {
+    public static final double kClimberP0 = 2;
+    public static final double kClimberI0 = 0;
+    public static final double kClimberD0 = 1;
+    public static final double kClimberkG0 = 0;
+    public static final double kClimberkV0 = 2;
+
+    public static final double kClimberP1 = 2;
+    public static final double kClimberI1 = 0;
+    public static final double kClimberD1 = 1;
+    public static final double kClimberkG1 = 0.5;
+    public static final double kClimberkV1 = 2;
+
+    public static final double KGearRatio = 126;
+    public static final Angle kClimberExtendedSetpoint = Degrees.of(190);
+    public static final Angle kClimberAtCageSetpoint = Degrees.of(170);
+    public static final Angle kClimberRetractedSetpoint = Degrees.of(90);
+    public static final Angle kClimberSensorError = Degrees.of(5);
```

Comment:
With the gear ratio so high, we should be able to get closer than this. Try 1-2 degrees